base/applications/drwtsn32/lang/pl-PL.rc

```diff
@@ -0,0 +1,11 @@
+/*
+ * PROJECT:     Dr. Watson crash reporter
+ * LICENSE:     GPL-2.0+ (https://spdx.org/licenses/GPL-2.0+)
+ * PURPOSE:     pl-PL resource file
+ */
+
```

Comment:
As in all language files, your file misses the obligatory line:
```
LANGUAGE LANG_POLISH, SUBLANG_DEFAULT
```
(and I know, the english language file misses its own too:
```
LANGUAGE LANG_ENGLISH, SUBLANG_ENGLISH_US
```
)